roles/eda/templates/eda.configmap.yaml.j2

```diff
@@ -24,7 +24,7 @@ data:
   # EDA Server
   EDA_ALLOWED_HOSTS: "['*']"
   EDA_CSRF_TRUSTED_ORIGINS: "http://{{ api_server_name }}:{{ api_nginx_port }}"
-  EDA_MQ_HOST: "{{ ansible_operator_meta.name }}-redis-svc"
+  #EDA_MQ_HOST: "{{ ansible_operator_meta.name }}-redis-svc"
```

Comment:
Is this intentional? should it just be removed?

Reply:
It was intentional during testing but I have removed it as we move towards a final product.